File: Back/src/main/java/fr/uga/miage/m1/DTO/Achat.java
package fr.uga.miage.m1.DTO;


import java.util.List;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class Achat {

    long numAchat;

    
    long nbPlace;

    
    Boolean achatValidee;

    Utilisateur utilisateur;
    
    List<EtapeAchat> etapeAchat;
}

File: festiCarFront/src/models/Achat.ts

import { EtapeAchat } from "./EtapeAchat";
import { Utilisateur } from "./Utilisateur";


export interface Achat {
    numAchat: number;
    nbPlace: number;
    achatValidee: boolean;
    utilisateur: Utilisateur;
    etapeAchat: EtapeAchat[];
  }